lib/utils/log-file.js

```diff
@@ -48,7 +48,11 @@ class LogFiles {
     this.#endStream()
   }
 
-  load ({ path, logsMax = Infinity, timing } = {}) {
+  load ({ command, path, logsMax = Infinity, timing } = {}) {
+    if (['completion'].includes(command)) {
```

Comment:
is this an optimization to allow for more to be added later?

Reply:
Doing the command check here is the best way to do this, but making it an array is just to allow more to be added in the future.